client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -178,9 +175,11 @@ const MarketplaceThankYou = ( { productSlug }: { productSlug: string } ) => {
 		| { action_links?: { Settings?: string }; name?: string };
 
 	const fallbackSetupUrl = wpComPluginData?.setup_url && siteAdminUrl + wpComPluginData?.setup_url;
+	const managePluginsUrl = hasManagePluginsFeature
+		? `${ siteAdminUrl }plugins.php`
+		: `/plugins/manage/${ siteSlug } `;
```

Comment:
Let's avoid using `plugins/manage`. We 've been using `/plugins.php` in case we didn't have a specific WP Admin URL (action_link). Can we replace it with `/plugins/pluginSlug`?

Reply:
`wp-admin/plugins.php` is only available for sites with the `MANAGE_FEATURE`, so we need to redirect to something else for the rest of sites (e.g. Atomic sites with a Starter plan). See 837-gh-Automattic/wpcomsh

Reply:
Yep, got that part. That's why I suggest to redirect to Calypso `/plugins/pluginSlug/siteId`

Reply:
Oh sorry, I read your suggestion too fast and thought you were suggesting to revert the changes 🙂 Yes, that could work!